tests/lib/common.py

```diff
@@ -63,3 +65,64 @@ def wait_for_result(func, *args, matcher=simple_matcher(True), attempts=20,
     logger.error(out)
 
     raise Exception("Timed out waiting for result")
+
+
+def execute(command, capture=False):
+    """A helper util to excute `command`.
+
+    The stdout and stderr are redirected to the logging module. You can
+    optionally catpure it by setting `capture` to True.
+    stderr is logged as a warning as it is up to the caller to raise any
+    actual errors from the RC code.
+
+    Returns a tuple of (rc code, output), where output is a dict with stdout
+    and stderr if capture is True.
+    """
+    process = subprocess.Popen(
+        command,
+        shell=True,
```

Comment:
Until now, I think we did not use `shell=True` for calling commands.

Reply:
No, and it might not be necessary, but it simplifies things and generally works as expected since we're only really supporting running this on linux.